docs/reference/func_config_git_set.md

```diff
@@ -26,7 +26,7 @@ func config git set
       --gh-webhook-secret string   GitHub Webhook Secret used for payload validation. If not specified, it will be generated automatically.
   -t, --git-branch string          Git revision (branch) to be used when deploying via the Git repository ($FUNC_GIT_BRANCH)
   -d, --git-dir string             Directory in the Git repository containing the function (default is the root) ($FUNC_GIT_DIR)
-      --git-provider string        The type of the Git platform provider to setup webhook. This value is usually automatically generated from input URL, use this parameter to override this setting. Currently supported providers are "github".
+      --git-provider string        The type of the Git platform provider to setup webhook. This value is usually automatically generated from input URL, use this parameter to override this setting. Currently supported providers are "github" and "gitlab".
```

Comment:
I suspect this is from @matejvasek's latest gitlab PR which wasn't reflected in https://github.com/knative/func/pull/1769

Reply:
Yep, these are easy to forget.  It would be nice to have a CI task which regens the docs and checks if git shows a diff or some other such failsafe